tfe/resource_tfe_organization.go

```diff
@@ -124,6 +137,23 @@ func resourceTFEOrganizationRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("cost_estimation_enabled", org.CostEstimationEnabled)
 	d.Set("send_passing_statuses_for_untriggered_speculative_plans", org.SendPassingStatusesForUntriggeredSpeculativePlans)
 
+	// update admin settings
+	log.Printf("[DEBUG] Read admin settings of organization: %s", d.Id())
+	adminOrg, err := tfeClient.Admin.Organizations.Read(ctx, d.Id())
+	if err != nil {
+		return err
+	}
+	var adminSettings []interface{}
+
+	if adminOrg.WorkspaceLimit != nil && *adminOrg.WorkspaceLimit != 0 {
+		adminSettings = append(adminSettings, map[string]interface{}{
+			"workspace_limit": *adminOrg.WorkspaceLimit,
+		})
+	}
+	if err := d.Set("admin_settings", adminSettings); err != nil {
+		return fmt.Errorf("error setting admin settings for organization %s: %w", d.Id(), err)
```

Comment:
💅 For consistency with other error messages in this file, suggest to capitalize the first E in error:

```suggestion
		return fmt.Errorf("Error setting admin settings for organization %s: %w", d.Id(), err)
```